semperfiwebdesign/all-in-one-seo-pack
# Enable Schema.org by default

Issue #2868

## Proposed changes
This changes the default setting for Use Schema.org Markup to Enabled.  This only affects new installs.

## Types of changes
- Improves existing functionality
- Improves existing code

## Checklist

## Testing instructions
- Install this on a fresh site and confirm that the Use Schema option is enabled by default
- Install this on an existing site and reset the General Settings to default and confirm that the Use Schema option is enabled by default
- Install this on an existing site with the current live version and where Use Schema is disabled, confirm that the option is not changed
